src/nwbinspector/checks/ecephys.py

```diff
@@ -79,3 +79,17 @@ def check_spike_times_not_in_unobserved_interval(units_table: Units, nunits: int
                     "observed intervals."
                 )
             )
+
+
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=Units)
+def check_ascending_spike_times(units_table: Units):
+    """Check that the values in the timestamps array are strictly increasing."""
+    if "spike_times" not in units_table:
+        return
+    differences = np.diff(np.asarray(units_table["spike_times"].target.data[:]))
```

Comment:
I do believe for this one we will have to iterate over the units identified via the API convenience functions, _i.e._,

```python
for unit_index in range(len(units_table.id)):
    spike_times_per_unit = units_table.get_unit_spike_times(index=unit_index)
    differences_per_unit = np.diff(spike_times_per_unit)
```

Reason for this will be clear when you add tests of this function; creating a simple `Units` table with two units, each having a few 'good' (monotone increasing) spike times, then looking at the value of  `units_table["spike_times"].target.data[:]` will reveal quite a lot about how NWB structures the table

Reply:
This would be simpler:

```python
for spike_times_per_unit in units_table["spike_times"]:
    differences_per_unit = np.diff(spike_times_per_unit)
```